core/control-plane/contract-core/src/test/java/org/eclipse/edc/connector/contract/offer/ContractOfferResolverImplIntegrationTest.java

```diff
@@ -107,6 +112,42 @@ void shouldLimitResult_withHeterogenousChunks() {
         verify(policyStore).findById("contract");
     }
 
+    private static Stream<Arguments> ranges() {
+        return Stream.of(
+                Arguments.of(0, 12),
+                Arguments.of(8, 14),
+                Arguments.of(0, 999),
+                Arguments.of(4, 888),
+                Arguments.of(3, 20),
+                Arguments.of(23, 25)
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("ranges")
```

Comment:
I'd rather you use `@CsvSource`, because then at least the test values are immediately visible, or `@ArgumentSource`, because that uses a strongly typed provider rather thant relying solely on the method name.
Also, the method naming should follow the `should_when_then` pattern, separated by an underscore, which increases legibility quite a bit.

Reply:
thx, Paul.. it's fixed.